test/realm_test.dart

```diff
@@ -497,6 +497,33 @@ Future<void> main([List<String>? args]) async {
       expect(cars.isEmpty, true);
     });
 
+    test('Results.query() isEmpty', () {
```

Comment:
perhaps  `Results.all() isEmpty` test should be named `Results isEmpty` and then we don't need this one. But it does not hurt to have it. Feel free to leave it if you would like.

Reply:
Results.all() isEmpty was existing and I created the new one similar to it. But you are right we have both test cases in the new test. I will remove the first test and will rename the second test.

Reply:
Actually, both tests are a little bit different. It is good to have them both. Just will rename them.